table-store/src/main/java/com/scalar/dl/tablestore/client/tool/TableStoreCommandLine.java

```diff
@@ -0,0 +1,79 @@
+package com.scalar.dl.tablestore.client.tool;
+
+import static picocli.CommandLine.Model.UsageMessageSpec.SECTION_KEY_COMMAND_LIST;
+import static picocli.CommandLine.Model.UsageMessageSpec.SECTION_KEY_COMMAND_LIST_HEADING;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import com.scalar.dl.client.tool.CertificateRegistration;
+import com.scalar.dl.client.tool.CommandGroupRenderer;
+import com.scalar.dl.client.tool.ContractsListing;
+import com.scalar.dl.client.tool.SecretRegistration;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.HelpCommand;
+
+@Command(
+    name = "scalardl-table-store",
+    subcommands = {
+      CertificateRegistration.class,
+      ContractsListing.class,
+      ContractsRegistration.class,
+      HelpCommand.class,
+      LedgerValidation.class,
+      SecretRegistration.class,
+      StatementExecution.class,
+    },
+    description = {"These are ScalarDL Table Store commands used in various situations:"})
```

Comment:
Just to confirm, are we officially going to capitalize `Table Store` as a proper noun? If not, I think we should lowercase it here.

Reply:
That's a good point. I just used it, similar to `ScalarDB Cluster`, but let me discuss it with Hiro.